ca/server.go

```diff
@@ -69,6 +69,30 @@ func (s *Server) SetReconciliationRetryInterval(reconciliationRetryInterval time
 	s.reconciliationRetryInterval = reconciliationRetryInterval
 }
 
+// GetUnlockKey is responsible for returning the current unlock key used for encrypting TLS private keys and
+// other at rest data.  Access to this RPC call should only be allowed via mutual TLS from managers.
+func (s *Server) GetUnlockKey(ctx context.Context, request *api.GetUnlockKeyRequest) (*api.GetUnlockKeyResponse, error) {
+	log.G(ctx).WithFields(logrus.Fields{
+		"method": "GetUnlockKey",
+	})
+
+	resp := api.GetUnlockKeyResponse{}
+	s.store.View(func(tx store.ReadTx) {
+		cluster := store.GetCluster(tx, s.securityConfig.ClientTLSCreds.Organization())
```

Comment:
IRL discussion:  We made this change, but it can cause a delay between when the cluster is updated to rotate the key, and when `GetUnlockKey` reflects this new key.  To avoid having `GetUnlockKey` return an old key after a cluster is updated, we'd either have to wait some arbitrary amount of time after rotating the manager key to call `GetUnlockKey`, or we just have to call `View` here in this `GetUnlockKey` function to ensure we always return the latest key.  I've added a comment above this chunk of code to that effect.